spec/factories/approvals.rb

```diff
@@ -1,10 +1,10 @@
 FactoryGirl.define do
```

Comment:
Oops, looks like we need it to make some associations in other factories work easier.